Userland/Libraries/LibCore/MimeData.cpp

```diff
@@ -166,6 +166,17 @@ Optional<StringView> guess_mime_type_based_on_sniffed_bytes(ReadonlyBytes bytes)
     return {};
 }
 
+Optional<StringView> get_description_from_mime_type(StringView mime_name)
```

Comment:
Commit message, again: _the `get_description_from_mime_type` is_ -> _the `get_description_from_mime_type` function is_

Reply:
I'm sorry, I'm really unable to speak English correctly these days.